### DISCUSSION TITLE
Automatically comment on latest announcement post

### DISCUSSION THREAD

**Discussion by kenn1ld** [3urobeat/steam-comment-service-bot#254] [Q&A]
Is it possible to configure this to automatically comment on the latest news post? https://store.steampowered.com/news/

**Comment by 3urobeat** [comment]
Hey!  
Automatic request in general are not supported,  they always have to be requested by someone or something.  

This could, for example, be a plugin which regularly checks a page for new posts and then starts a request for it.
